examples/pytorch/CaptumExample/Titanic_Captum_Interpret.py

```diff
@@ -0,0 +1,394 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+"""
+Getting started with Captum - Titanic Data Analysis
+"""
+# Initial imports
+import numpy as np
+import torch
+from captum.attr import IntegratedGradients
+from captum.attr import LayerConductance
+from captum.attr import NeuronConductance
+import matplotlib.pyplot as plt
+from scipy import stats
+import pandas as pd
+import mlflow
+from prettytable import PrettyTable
+from mlflow.utils.autologging_utils import try_mlflow_log
+import tempfile
+import shutil
+import os
+from argparse import ArgumentParser
+import torch.nn as nn
+from torchtext.utils import download_from_url
+
+mlflow.start_run(run_name="Titanic_Captum_mlflow")
+
+
+def TitanicDataset():
```

Comment:
```suggestion
def get_titanic():
```

Can we use the snake case here instead of the camel case which is ususally used for a class?

Reply:
@harupy Done.